### PR TITLE
Removing on-the-fly CTU functionality as it is not supported by clang

### PR DESCRIPTION
**Pull request by dkrupp**

### CODE REVIEW

File: tests/functional/ctu/test_ctu.py [hunk 1]
```diff
@@ -66,98 +66,74 @@ def tearDown(self):
...
 
-    def __do_ctu_all(self, reparse):
+    def __do_ctu_all(self):
         """ Execute a full CTU run. """
 
         cmd = [self._codechecker_cmd, 'analyze', '-o', self.report_dir,
-               '--analyzers', 'clangsa', '--ctu-all']
-        if reparse:
-            cmd.append('--ctu-on-the-fly')
+               '--analyzers', 'clangsa', '--ctu-all','--verbose','debug']
```
Comment (csordasmarton) [inline tests/functional/ctu/test_ctu.py]: Space is required after `,`. Please fix other Codacy warnings too.

File: tests/functional/ctu/test_ctu.py [hunk 2]
```diff
@@ -66,98 +66,74 @@ def tearDown(self):
...
 
-    def __do_ctu_analyze(self, reparse):
+    def __do_ctu_analyze(self):
         """ Execute CTU analyze phase. """
 
         cmd = [self._codechecker_cmd, 'analyze', '-o', self.report_dir,
-               '--analyzers', 'clangsa', '--ctu-analyze']
-        if reparse:
-            cmd.append('--ctu-on-the-fly')
+               '--analyzers', 'clangsa', '--ctu-analyze','--verbose','debug']
```
Comment (csordasmarton) [inline tests/functional/ctu/test_ctu.py]: Space is required after `,`

File: libcodechecker/analyze/analyzers/config_handler_clangsa.py [hunk 3]
```diff
@@ -20,8 +20,7 @@ class ClangSAConfigHandler(config_handler.AnalyzerConfigHandler):
     def __init__(self):
         super(ClangSAConfigHandler, self).__init__()
         self.__checker_configs = []
-        self.__ctu_dir = ''
-        self.__ctu_in_memory = False
+        self.__ctu_dir = ''
```
Comment (csordasmarton) [inline libcodechecker/analyze/analyzers/config_handler_clangsa.py]: Remove trailing white spaces.

**Comment by gyorb** [comment]
clang6 does not support this feature yet, if it will be introduced in clang6 or later we should review if we want to re implement is as it was.

**Pull request merged**